vizro-core/pyproject.toml

```
@@ -28,7 +28,7 @@ dependencies = [
   "autoflake",
   "packaging"
 ]
-description = "Vizro is a package to facilitate visual analytics."
+description = "Vizro is a low-code toolkit for building high-quality data visualization apps."
```

```suggestion
description = "Vizro is a low-code framework for building high-quality data visualization apps."
```

Since this description specifically concerns the vizro core package rather than the repo as a whole.